idea/testData/intentions/convertToBlockBody/implicitlyNonUnitFun2.kt.after

```diff
@@ -0,0 +1,5 @@
+// WITH_RUNTIME
+
+fun foo(): Array<out String?>? {
```

Comment:
The `?` inside `Array` is there because FIR does not seem to have any better knowledge. I don't think it's right to suppress it in the intention. We probably need some change on the FIR side to improve this behavior.

Reply:
This may happens because of missing external annotations in test infrastructure. It should be interesting to check how it behave in "real" IJ IDEA with external annotations enabled. They should provide nullability annotations for JDK methods

Reply:
Dev instance with FIR plugin behaves the same :(